### ISSUE TITLE
config.ts: Environment variable names error message not set properly

### ISSUE THREAD

**Issue by brein62** [CS3219-AY2425S1/cs3219-ay2425s1-project-g36#43]
In `./user-service/utils/config.ts`, the error messages do not reflect the right variable:

```
if (!process.env.USER_SERVICE_MONGODB_URI) {
    throw new Error('USER_SERVICE_MONGODB_URL is not defined in the environment variables');
}

if (!process.env.BLIST_MONGODB_URI) {
    throw new Error('BLIST_MONGO_URL is not defined in the environment variables');
}
```

Need to change `USER_SERVICE_MONGODB_URL` to `USER_SERVICE_MONGODB_URI` and `BLIST_MONGO_URL` to `BLIST_MONGODB_URI`

**Comment by brein62** [comment]
alternatively, we could use the given user service...

**Comment by brein62** [comment]
Fixed by #44
